doc/1/controllers/document/update/index.md

```diff
@@ -0,0 +1,125 @@
+---
+code: true
+type: page
+title: update
+description: Updates a document
+---
+
+# update
+
+Updates a document.
+
+---
+
+:::: tabs
+::: tab Java
+
+## Arguments
+ 
+```java
+public CompletableFuture<ConcurrentHashMap<String, Object>> update(
+      final String index,
+      final String collection,
+      final String id,
+      final ConcurrentHashMap<String, Object> document)
+throws NotConnectedException, InternalException
+
+public CompletableFuture<ConcurrentHashMap<String, Object>> update(
+      final String index,
+      final String collection,
+      final String id,
+      final ConcurrentHashMap<String, Object> document,
+      final UpdateOptions options)
+throws NotConnectedException, InternalException
+```
+
+| Arguments          | Type                                         | Description                       |
+| ------------------ | -------------------------------------------- | --------------------------------- |
+| `index`            | <pre>String</pre>                            | Index                             |
+| `collection`       | <pre>String</pre>                            | Collection                        |
+| `id        `       | <pre>String</pre>                            | Document ID                        |
+| `document`         | <pre>ConcurrentHashMap<String, Object></pre> | Partial document content |
+| `options`          | <pre>UpdateOptions</pre><br>(`null`)         | Optional parameters               |
```

Comment:
```suggestion
| `waitForRefresh`   | <pre>Boolean</pre>                           | If set to `true`, Kuzzle will wait for the persistence layer to finish indexing|
| `retryOnConflict`  | <pre>Integer</pre>                           | The number of times the database layer should retry in case of version conflict |
| `source`           | <pre>Boolean</pre>                           | If true, returns the updated document inside the response |
```